### ISSUE TITLE
Possible bug on world load

### ISSUE THREAD

**Issue by RoboticSith** [MrJake222/AUNIS#204]
I am working on an adventure style map using AUNIS and several other mods.  I recently updated from 1.4.1-beta-hotfix2 to 1.9.9-beta so I could use the universe style gates.  Other than a few occasional rendering glitches when I loaded the given dimensions, they all worked correctly.  Starting about 1 week ago however, the world started to only load with about a 25% success rate (dumping straight back to the desktop when it failed - no crashlog since it never loaded).  Now, the world refuses to load ever.  The launcher log (attached - line 16858) shows that it is failing to load the block data for the milkyway base block.  It then gets stuck in a loop and eventually crashes the launcher, and the game (again without a crashlog).  I have a total of 2 Milky-way gates & 4/6 Universe gates.  All are connected to infinite energy source blocks.  I am currently in the overworld and this particular gate has a glyph crystal along with a Milkyway address and Universe address crystal.  I've tried changing the amount of memory allocated to the game when I try to load it, but nothing has worked so far.  Thanks in advance.

**Comment by RoboticSith** [comment]
Attachment failed to upload with original post



[latest.zip](https://github.com/MrJake222/AUNIS/files/4742492/latest.zip)

**Comment by RoboticSith** [comment]
I used MCedit to sequentially replace parts of the gates (in all dimensions) with air and I eventually started getting a camo error in the main log.  I then went through and systematically removed camo'd blocks and now it is getting stuck in a loop calling updatePowerTier perpetually until the launcher and game crash.  Logs are attached.  I'm going to try a full removal of all the gates and see if I can get the world to even load.
[latest-camoerror.zip](https://github.com/MrJake222/AUNIS/files/4862003/latest-camoerror.zip)
[latest-updatePowerTier loop.zip](https://github.com/MrJake222/AUNIS/files/4862005/latest-updatePowerTier.loop.zip)

**Comment by RoboticSith** [comment]
So I went though and removed all blocks from the gates (and a ring platform I had) and it was still giving the same looping error and not allowing the world to load.  The only items remaining from AUNIS were various blocks and items (configured Universe Remotes) in chests (I don't know how to edit contents in MCedit) that I was using as a repository for map making stuff.  Even with all placed blocks removed, the error was still occurring.  The only way that I was able to get the world to finally load was to completely disable the mod.
